src/platforms/common/configuration/options.mdx

```diff
@@ -113,7 +113,7 @@ Grouping in Sentry is different for events with stack traces and without. As a r
 
 </ConfigKey>
 
-<ConfigKey name="send-default-pii" supported={["dotnet", "javascript", "node", "react-native", "python", "java", "php", "rust"]}>
+<ConfigKey name="send-default-pii" supported={["apple", "dotnet", "javascript", "node", "react-native", "python", "java", "php", "rust"]}>
```

Comment:
not related to this PR but:
we'd need to go thru all the options (options.mdx) and check 1 by 1 for each mobile platform, I suspect a few of them are missing, I don't recall adding new options here often but I do recall adding on code :P

Reply:
https://github.com/getsentry/sentry-docs/issues/2985 so we don't forget